cms/static/js/views/feedback.js

```diff
@@ -1,146 +0,0 @@
-define(["jquery", "underscore", "underscore.string", "backbone", "js/utils/templates"],
```

Comment:
@andy-armstrong I don't know what happened here. I have used `git mv`. I am looking into this. Thanks!